index.css

```diff
@@ -5,12 +5,34 @@
     padding: 0;
 
 }
+::-webkit-scrollbar {
+    width: 8px;
+  }
+  ::-webkit-scrollbar-track {
+    box-shadow: inset 0 0 5px grey; 
+    
+  }
+  ::-webkit-scrollbar-thumb:hover {
+    background:var(--primary-btn); 
 
+  }
+.header{
+display: flex;
+flex-direction: column;
+position: relative;
+}
 .header-nav {
     display: flex;
     padding: 1rem 1rem 0.5rem 2rem;
     justify-content: space-between;
     align-items: center;
+    width: 81rem;
+    height: 5rem;
+    position: sticky;
+    top:0;
+    left:0;
+    z-index:999;
```

Comment:
why such hight z-index value?, z-index of 2 or 3 would be more than enough for navbar